packages/bot/src/stores/flyout-store.js

```diff
@@ -1,11 +1,19 @@
 /* eslint-disable no-underscore-dangle */
 import { observable, action } from 'mobx';
```

Comment:
Please fix the spacing for these imports:

```JavaScript
import my_module       from 'file.js';
import my_other_module from 'file2.js';
```

Reply:
done